spec.bs

```diff
@@ -464,6 +464,16 @@ The <dfn attribute for=HTMLFencedFrameElement>config</dfn> IDL attribute getter
        Note: This holds because when the element has been removed from the DOM, its removal steps
        immediately destroy the [=fenced navigable container/fenced navigable=].
 
+  1. Let |instance| be [=this=]'s [=relevant global object=]'s [=Window/browsing context=]'s
+     [=browsing context/fenced frame config instance=].
+
+  1. If |instance| is not null:
+     1. If |instance|'s [=fenced frame config instance/has disabled untrusted network=] is true,
+        then return early.
+
+     Note: Embedder-initiated navigations of fenced frame roots are not allowed after the
+     embedder's network has been disabled.
```

Comment:
This is stating the obvious: https://google.github.io/styleguide/cppguide.html#Implementation_Comment_Donts. It's better to describe _why_ this is important.

Reply:
I'll just delete it then. The reason I left the note was to clarify that this was about the embedder's network being disabled (i.e. it's a nested fenced frame), but maybe this is obvious.